src/v0/destinations/attentive_tag/transform.js

```diff
@@ -137,7 +137,7 @@ const trackResponseBuilder = (message, { Config }) => {
       payload = constructPayload(message, mappingConfig[ConfigCategory.TRACK.name]);
       endpoint = ConfigCategory.TRACK.endpoint;
       payload.type = get(message, 'event');
-      if (!getPropertiesKeyValidation(payload)) {
+      if (!arePropertiesValid(payload.properties)) {
         throw new InstrumentationError(
           '[Attentive Tag]:The event name contains characters which is not allowed',
```

Comment:
The above if condition to verify properties and error message being thrown doesn't seem to be in sync. Are we sure the error message is correct?

Reply:
yes, this appropriate if you see the body of `arePropertiesValid` it is checking for special characters

Reply:
but we are checking for all properties and not only for event name or are we ?

Reply:
got it, let me change